synapse/handlers/federation_event.py

```diff
@@ -1312,6 +1310,9 @@ def prep(event: EventBase) -> Optional[Tuple[EventBase, EventContext]]:
                         return None
                     auth.append(ae)
 
+                # we're not bothering about room state, so flag the event as an outlier.
+                event.internal_metadata.outlier = True
```

Comment:
Should the docstring of `_auth_and_persist_outliers` (or `_auth_and_persist_outliers_inner`) be updated to note that it will mark events as outliers?